app/component/ItineraryTab.js

```diff
@@ -132,6 +132,15 @@ class ItineraryTab extends React.Component {
       return null;
     }
 
+    if (itinerary.fares === null && config.URL.FARES) {
+      fetchFares(itinerary, config.URL.FARES)
```

Comment:
I looked into why the `POST`ed itineraries are so large, and according to the sample that Leonard gave me, it seems that the JSON payload sent to `fares-service` contains parts of the GraphQL schema as well.

Is there a simple and straightforward way to get the itinerary without the GraphQL schema bits? It's not a real problem though, just an optimisation.

Reply:
Shouldn't be hard to remove them. Can you point towards the unneeded properties?

Reply:
I believe @derhuerst thought about the keys starting with the double underscore. Inside each leg id, fragments, fragmentOwner. I'm about to filter these out!